src/main/java/seedu/address/model/appointment/Time.java

```diff
@@ -20,8 +21,10 @@
     // Closes at 10pm
     public static final LocalTime CLOSING_TIME = LocalTime.of(22, 0);
 
-    public static final String MESSAGE_CONSTRAINTS = "The time should fall within the opening hours: "
-            + "from " + OPENING_TIME.format(TIME_FORMAT) + " to " + CLOSING_TIME.format(TIME_FORMAT);
+    public static final String MESSAGE_CONSTRAINTS = "The appointment time period should fall within "
+            + "the opening hours: from " + OPENING_TIME.format(TIME_FORMAT)
+            + " to " + CLOSING_TIME.format(TIME_FORMAT)
+            + "\nThe appointment duration of " + DEFAULT_DURATION.toHours() + "hours should also be considered.";
```

Comment:
If the default duration is 1 hour, should "hour" instead of "hours" be used?

Reply:
Oh I missed that out! I'll update it to "hour". Thanks :+1